package.json

```diff
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "manifest": "script/build.sh",
-        "build": "npm test && npm run manifest && next build",
+        "build": "NODE_OPTIONS=--openssl-legacy-provider npm test && npm run manifest && NODE_OPTIONS=--openssl-legacy-provider next build",
```

Comment:
What is this change?

Reply:
Fixes the build in Vercel which now requires node 18